Gemfile

```diff
@@ -100,7 +100,7 @@ end
 
 group :cucumber, :test do
   gem "cucumber"
-  gem "cucumber-rails", require: false
+  gem "cucumber-rails", require: false, git: "https://github.com/cucumber/cucumber-rails.git", ref: "5bfb4219abc50a5d1c8b2fa0a04c441bbc6e5288"
```

Comment:
Can we add a comment to explain this (as per: https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html#gemfile-organisation)